linux_docker_resources/entry_point.sh

```diff
@@ -23,18 +23,17 @@ echo "done."
 # We only attempt to install Connext on amd64
 if [ "${ARCH}" != "aarch64" ]; then
     IGNORE_CONNEXTDDS=""
-    IGNORE_CONNEXTCPP=""
     ignore_rwm_seen="false"
     for arg in ${CI_ARGS} ; do
         case $arg in
             ("--ignore-rmw") ignore_rmw_seen="true" ;;
             ("-"*) ignore_rmw_seen="false" ;;
-            (*) if [ $ignore_rmw_seen = "true" ] ; then [ $arg = "rmw_connextdds" ] && IGNORE_CONNEXTDDS="true" ; [ $arg = "rmw_connext_cpp" ] && IGNORE_CONNEXTCPP="true" ; fi
+            (*) if [ $ignore_rmw_seen = "true" ] ; then [ $arg = "rmw_connextdds" ] && IGNORE_CONNEXTDDS="true" && break ; fi
```

Comment:
> Why add a break here where there wasn't one before? Is it just because once you've confirmed this condition there is no longer a need to parse arguments?

Yes, that's exactly it.  We are looking for one and only one thing, so we can get out early.